core/GlobalState.h

```diff
@@ -73,20 +73,20 @@ class GlobalState final {
     SymbolRef enterStaticFieldSymbol(Loc loc, SymbolRef owner, NameRef name);
     ArgInfo &enterMethodArgumentSymbol(Loc loc, SymbolRef owner, NameRef name);
 
-    SymbolRef lookupSymbol(SymbolRef owner, NameRef name) {
+    SymbolRef lookupSymbol(SymbolRef owner, NameRef name) const {
```

Comment:
Many pieces of this PR can be landed separately, could we extract them?

Reply:
Yup. This + the parseArgs change can be landed separately.